analyses/survival-analysis/survival-analysis_tp53_telomerase.Rmd

```diff
@@ -75,16 +75,28 @@ tel_stranded_file <- file.path(telomerase_dir,
 #### Import and process the metadata
 
 ```{r Read in metadata}
+# Will remove the below patients because I do not know if they had total or partial resections until we check with Jenn Mason
+
+# 7316-116 has Biopsy only;Gross/Near total resection
```

Comment:
If we may need to come back to this, we can make it more reproducible with an array - 

tbd_samples <- c("7316-116", "7316-182")

and then see comment on line 97

Reply:
I have gotten some updates on this yesterday, so will add this to the code

Reply:
Just created this ticket to track: https://github.com/PediatricOpenTargets/ticket-tracker/issues/298